research/metrics/v1.7.0/index.md

```diff
@@ -0,0 +1,160 @@
+
+* Gleckler, P. J., Taylor, K. E., and Doutriaux, C.: Performance metrics for climate models, J. Geophys. Res., 113, D06104, https://doi.org/10.1029/2007jd008972, **2008**. 
+
+
+---
+
+Questions or comments about the PCMDI Simulation Summaries should be sent to the [PMP team](mailto:pcmdi-metrics@llnl.gov).
+
+The efforts of the authors are supported by the [Regional and Global Climate Modeling Program][RGMA] of the [United States Department of Energy's Office of Science][DOEOS]. This work is funded by the Climate and Environmental Sciences Division of the DOE Office of Science and is performed under the auspices of the U.S. Department of Energy by [Lawrence Livermore National Laboratory][LLNL] under contract DE-AC52-07NA27344. LLNL-WEB-812310
+
+[latest]: {{site.baseurl}}/research/metrics/
+
+[v1.7.2]: {{site.baseurl}}/research/metrics/v1.6.0
+[v1.6.2]: {{site.baseurl}}/research/metrics/v1.6.2
+[v1.6.1]: {{site.baseurl}}/research/metrics/v1.6.1
+[v1.6.0]: {{site.baseurl}}/research/metrics/v1.6.0
+[v1.5.1]: {{site.baseurl}}/research/metrics/v1.5.1
+[v1.5.0]: {{site.baseurl}}/research/metrics/v1.5.0
+[v1.4.1]: {{site.baseurl}}/research/metrics/v1.4.1
+[v1.4.0]: {{site.baseurl}}/research/metrics/v1.4.0
+[v1.3.2]: {{site.baseurl}}/research/metrics/v1.3.2
+[v1.3.1]: {{site.baseurl}}/research/metrics/v1.3.1
+[v1.3.0]: {{site.baseurl}}/research/metrics/v1.3.0
+[v1.2.0]: {{site.baseurl}}/research/metrics/v1.2.0
+[v1.1.0]: {{site.baseurl}}/research/metrics/v1.1.0
+[v1.0.0]: {{site.baseurl}}/research/metrics/v1.0.0
+[v1.0.0-beta]: {{site.baseurl}}/research/metrics/v1.0.0-beta
+[v1.0.0-alpha]: {{site.baseurl}}/research/metrics/v1.0.0-alpha
+
+[description_mean_clim]: mean_clim
+[description_variability]: variability_modes
+[description_monsoon]: monsoon
+[description_mjo]: mjo
+[description_enso]: enso
+[description_precip]: precip
+
+[lee2019]: https://link.springer.com/article/10.1007/s00382-018-4355-4
+[lee2021]: https://journals.ametsoc.org/view/journals/clim/34/17/JCLI-D-20-0832.1.xml
+[sperber2004]: https://doi.org/10.1007/s00382-014-2099-3
+[ahn2017]: https://doi.org/10.1007/s00382-017-3558-4
+[Bellenger2014]: https://doi.org/10.1007/s00382-013-1783-z
+[gleckler2008]: https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2007JD008972
+[taylor2001]: https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/2000JD900719
+[pendergrass2020]: https://doi.org/10.1175/BAMS-D-19-0318.1
+[doeworkshop2019]: https://climatemodeling.science.energy.gov/news/doe-hosts-precipitation-metrics-workshop
+
+[research]:{{site.baseurl}}/research
+[metrics]:{{site.baseurl}}/research/metrics/
+
+[mean_clim]: mean_clim
+[enso]: enso
+[variability_modes]: variability_modes
+[mjo]: mjo
+[monsoon]: monsoon
+[precip]: precip
+[sea_ice]: sea_ice
+[qbo]: qbo
```

Comment:
You will need to make pointers for these directories like for mean_clim and mjo.

Reply:
@mauzey1 thank you for reviewing. Pointers are now added via https://github.com/PCMDI/pcmdi.github.io/pull/558/commits/62aea32b0d743c1942a19e62ce4b7162368d05da